model/Hierarchy.php

```diff
@@ -11,30 +15,28 @@ class Hierarchy extends DataExtension {
 	
 	protected $markingFilter;
 	
-	/**
-	 * @var Int
-	 */
+	/** @var int */
```

Comment:
Not sure if there is a SilverStripe-specific standard, however the PSR-5 proposal currently allows it: https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#55-examples